testjson/testdata/summary/go-1-20-benchmark-panic

```diff
@@ -0,0 +1,8 @@
+
+=== Failed
+=== FAIL: github.com/go-logr/logr/benchmark BenchmarkDiscardLogInfoOneArg (unknown)
+=== RUN   BenchmarkDiscardLogInfoOneArg
+BenchmarkDiscardLogInfoOneArg
+BenchmarkDiscardLogInfoOneArg-36    	12695464	        91.33 ns/op
+
+DONE 1 tests, 1 failure in 0.000s
```

Comment:
Note that the default format is not informative enough when there was a panic: that panic output is not shown. 

Other formats don't have that problem:
```console
$ go run ./ --format=standard-verbose --raw-command cat testjson/testdata/input/go-1-20-panicked-benchmark.out
goos: linux
goarch: amd64
pkg: github.com/go-logr/logr/benchmark
cpu: Intel(R) Core(TM) i9-7980XE CPU @ 2.60GHz
=== RUN   BenchmarkDiscardLogInfoOneArg
BenchmarkDiscardLogInfoOneArg
BenchmarkDiscardLogInfoOneArg-36    	12695464	        91.33 ns/op
PASS
ok  	github.com/go-logr/logr/benchmark	1.265s
panic: fabricated panic
=== Failed
=== FAIL: github.com/go-logr/logr/benchmark BenchmarkDiscardLogInfoOneArg (unknown)
=== RUN   BenchmarkDiscardLogInfoOneArg
BenchmarkDiscardLogInfoOneArg
BenchmarkDiscardLogInfoOneArg-36    	12695464	        91.33 ns/op

DONE 1 tests, 1 failure in 0.001s
```

This is unrelated to this PR, I just noticed because the tests use the default format.